newtab.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <!-- viewport meta tag -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- pull in local jquery file -->
+    <script src="thirdparty/jquery-3.4.1.min.js" charset="utf-8"></script>
+    <!-- run app script -->
+    <script src="app.js" charset="utf-8"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>HomeTab</title>
+  </head>
+  <body>
+    <div class="grid-container">
```

Comment:
Although it does not change the performance on the code. I like always to try to make some sense divs. 

for example the div from line 26 could be an aside, or an article or any other type... just to make it easier to read.

Reply:
I think I've just gotten very used to using divs for everything, but readability is very important! I'll keep that in mind for future code.